packages/backend/src/tokens/request.ts

```diff
@@ -291,6 +308,63 @@ ${error.getFullMessage()}`,
     return signedOut(authenticateContext, reason, message);
   }
 
+  /**
+   * Determines if a handshake must occur to resolve a mismatch between the organization as specified
+   * by the URL (according to the options) and the actual active organization on the session.
+   *
+   * @returns {HandshakeState | SignedOutState | null} - The function can return the following:
+   *   - {HandshakeState}: If a handshake is needed to resolve the mismatched organization.
+   *   - {SignedOutState}: If a handshake is required but cannot be performed.
+   *   - {null}:           If no action is required.
+   */
+  function handleMaybeOrganizationSyncHandshake(
+    authenticateContext: AuthenticateContext,
+    auth: SignedInAuthObject,
+  ): HandshakeState | SignedOutState | null {
+    const toActivate = getOrganizationSyncTarget(authenticateContext.clerkUrl, options.organizationSyncOptions);
```

Comment:
```suggestion
    const organizationSyncTarget = getOrganizationSyncTarget(authenticateContext.clerkUrl, options.organizationSyncOptions);
```

@izaaklauer How about renaming this to organization target, it's confusing a bit from `toActivate` to `mustActivate` below those almost look the same thing.